src/index.js

```diff
@@ -1,20 +1,35 @@
-import _ from 'lodash';
-import printMe from './print.js';
 import './style.css';
 
-function component() {
-	const element = document.createElement('div');
-	const btn = document.createElement('button');
-  
-	element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-	element.classList.add('hello');
-
-	btn.innerHTML = 'Click me and check the console!';
-	btn.onclick = printMe;
-
-	element.appendChild(btn);
-  
-	return element;
-  }
-  
-  document.body.appendChild(component());
\ No newline at end of file
+window.addEventListener('load', () => {
+  const form = document.querySelector('.add-list');
+  const input = document.querySelector('#item-add');
+  const listEl = document.querySelector('.taskContainer');
+
+  form.addEventListener('submit', (e) => {
+    e.preventDefault();
+
+    const task = input.value;
+
+    const taskEl = document.createElement('div');
+    taskEl.classList.add('task');
+
+    const taskContent = document.createElement('div');
+    taskContent.classList.add('list-items');
+
+    taskEl.appendChild(taskContent);
```

Comment:
- [ ] Please 🙏🏽 write a function to iterate over the tasks array and populate an HTML list item element for each task, it is a project requirement.